app/controllers/search_controller.rb

```diff
@@ -1,8 +1,11 @@
 class SearchController < ApplicationController
   def index
-    
+
   end
 
   def show
+    @artist = Search.for(params[:q])[0]
+    # binding.pry
+    @related_artists = @artist.related_artists.sort_by {|artist| artist.followers["total"]}.reverse
```

Comment:
Line is too long. [99/80]<br>Space between { and | missing.<br>Space missing inside }.